packages/components/src/components/list-item-location/list-item-location.tsx

```diff
@@ -57,6 +63,8 @@ export class ListItemLocation {
      */
     @Event() listItemDidRender: EventEmitter;
 
+    @State() iconURLToRender; string;
```

Comment:
What is the purpose of using the `State`?

Reply:
It makes sure the component is re-rendered when it changes, in contrast to a private variable, which would do no such thing.

Reply:
Alright, good to know. Thanks